torch/_dynamo/variables/builtin.py

```diff
@@ -975,7 +975,7 @@ def _handle_insert_op_in_graph(self, tx: "InstructionTranslator", args, kwargs):
                 if fn is operator.truediv and isinstance(
                     args[0], variables.UnspecializedPythonVariable
                 ):
-                    args[0] = args[0].convert_to_constant(tx)
+                    args[0] = args[0].as_python_constant()
```

Comment:
This has been around since before dynamo was folded into pytorch.  `convert_to_constant` doesn't exist anywhere else in the codebase so I assume this was meant to be `as_python_constant` (and no tests seem to trigger this behavior).